client/src/components/TrailForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react"
+
+const TrailForm = (props) => {
+  const [trailRecord, setTrailRecord] = useState({
+    trailName: "",
+    trailLength: "",
+    trailLocation: "",
+    trailDescription: "",
+    estimateTime: "",
+  })
+
+  const handleChange = (event) => {
+    setTrailRecord({
+      ...trailRecord,
+      [event.currentTarget.name]: event.currentTarget.value,
+    })
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    props.addTrail(trailRecord)
+  }
+
+  return (
+    <div className="trail-form">
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="trailName">
+          Trail Name:
+          <input
+            id="trialName"
+            type="text"
+            name="trailName"
+            onChange={handleChange}
+            value={trailRecord.trailName}
+          />
+        </label>
+        <label htmlFor="trailLength">
```

Comment:
for readability lets add a line between each label!

Reply:
Unfortunately the changes in my coding theme that Fang made yesterday make it so the spaces auto-delete every time I save